gateway-js/src/index.ts

```diff
@@ -69,9 +69,11 @@ import { loadSupergraphSdlFromStorage } from './loadSupergraphSdlFromStorage';
 import { getServiceDefinitionsFromStorage } from './legacyLoadServicesFromStorage';
 import { buildComposedSchema } from '@apollo/query-planner';
 import { default as opentelemetry, SpanStatusCode } from "@opentelemetry/api";
-import {OpenTelemetrySpanNames} from "./utilities/opentelemetry";
+import { OpenTelemetrySpanNames } from "./utilities/opentelemetry";
```

Comment:
Should we just move `tracer` into this file instead of making it twice? (I think the otel stuff is smart enough that the two tracers share the same underlying smarts, but why make us have to do boilerplate on each file that wants it?)